packages/contracts/package.json

```diff
@@ -12,7 +12,7 @@
     "coverage": "yarn hardhat coverage",
     "export-abi": "yarn run hardhat export-abi",
     "clear-abi": "yarn run hardhat clear-abi",
-    "lint": "npx hardhat check",
+    "lint:sol": "npx hardhat check",
```

Comment:
Any reason for this change ?

Reply:
the docs specify `yarn run lint:sol` for solidity and `yarn run lint:ts` for typescript